publish.gradle

```diff
@@ -58,22 +58,13 @@ ext.configurePublish = { groupIdArg, artifactIdArg, descriptionArg ->
         }
     }
 
-    task zipMavenArtifacts(type: Zip) {
+    task renameSnapshotArtifacts(type: Exec) {
         dependsOn publish
 
-        from "${project.buildDir}/maven"
+        def renameScript = "${rootDir}/automation/taskcluster/rename_snapshot_artifacts.py"
+        def mavenBuildDir = "${project.buildDir}/maven"
 
-        include '**/*.aar'
-        include '**/*.jar'
-        include '**/*.pom'
-        include '**/*.md5'
-        include '**/*.sha1'
-
-        // Metadata is generated by maven.mozilla.org directly
-        exclude '**/*maven-metadata.xml*'
-
-        archiveName "target.maven.zip"
-        includeEmptyDirs = false
-        destinationDir(file("${project.buildDir}"))
+        commandLine 'python', renameScript, '--path', mavenBuildDir, '--timestamp', (project.hasProperty('timestamp') ? timestamp : '')
```

Comment:
Followup: I wonder if we should/if it's possible to port the logic of this script in gradle itself. Gradle know about the target files, it knows about the timestamp, and I believe it's suited to do operations on files.

No need to change this PR.

Reply:
Yeah, good point. I'm not too happy about hardcoding the paths here. If it would be possible, it would deprecate some of those checks too. Since gradle would name the files directly, we wouldn't need to worry about ensuring timestamps are similar or whatnot.